### PR TITLE
Improve doc links

### PR DESCRIPTION
**Pull request by ivy-lli**
- Add dropdown link list to doc le buttons (with news, release notes, migi notes)
image
- Add release notes link to the download overview page
image

### CODE REVIEW

**Comment by ivy-lli** [comment]
Maybe the evaluation of the additional doc links should come somewhere from the php backend.

File: src/app/pages/doc/overview.twig [hunk 1]
```diff
@@ -46,9 +45,20 @@
             <ul class="actions">
               {% for docLink in docLinksLE %}
                 <li>
-                  <a href="{{ docLink.url }}" class="button special min-width">
-                    <i class="si si-book"></i> {{ docLink.displayText }}
-                  </a>
+                  <span class="button special min-width">
```
Comment (alexsuter) [inline src/app/pages/doc/overview.twig:48]: why does the LTS version do not have this dropdown?

File: src/app/pages/doc/overview.twig [hunk 2]
```diff
@@ -46,9 +45,20 @@
...
+                    <a href="{{ docLink.url }}" >
+                      <i class="si si-book"></i> {{ docLink.displayText }}
+                    </a>
+                    <span class="doc-dropdown" ontouchstart="$(this).toggleClass('active-dropdown');">
```
Comment (alexsuter) [inline src/app/pages/doc/overview.twig:52]: I would add a method to the DocProvider which is accessible over RelaseInfo to evaluate this links.

File: src/app/pages/download/download.twig [hunk 3]
```diff
@@ -9,7 +9,14 @@
     <header>
       <h1>
         {{ headerTitle }}
-        <div  title="Latest Release Date" style="float: right; font-size: 18px;">{{ releaseDate }}</div>
+        <span class="download-infos">
```
Comment (alexsuter) [inline src/app/pages/download/download.twig:12]: All the right aligned stuff is not very responsive.
image

File: src/app/pages/download/DownloadAction.php [hunk 4]
```diff
@@ -237,6 +245,11 @@ public function archiveLink(): string
     return $this->releaseType->archiveLink($this->releaseInfo);
   }
 
+  public function releaseNotesLink(): string
+  {
+    return 'doc/' . $this->releaseInfo->minorVersion() . '/release-notes';
```
Comment (alexsuter) [inline src/app/pages/download/DownloadAction.php:250]: this should be somewhere available over releaseInfo - docProvider?

**Review by alexsuter — CHANGES_REQUESTED** [review]

File: src/app/pages/doc/overview.twig (hunk 1 above)
Comment (ivy-lli) [reply src/app/pages/doc/overview.twig:48]: Not done yet. I thought I better wait if you like this change...

File: src/app/pages/download/download.twig (hunk 3 above)
Comment (ivy-lli) [reply src/app/pages/download/download.twig:12]: I improved it a little bit, but yes I'm also not very pleased with the current result. But in general what do you think about this release notes link below the release date?

**Review by alexsuter — APPROVED** [review]

**Pull request merged**